app/code/Magento/UrlRewriteGraphQl/etc/schema.graphqls

```diff
@@ -3,7 +3,7 @@
 
 type EntityUrl @doc(description: "EntityUrl is an output object containing the `id`, `canonical_url`, and `type` attributes") {
```

Comment:
Would be nice if you could fix description of `EntityUrl` in scope of this PR. No need to have the list of fields there, because it causes discrepancies during refactoring (like now).